lib/widgets/AppBars.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+class ReturnHomeBar extends StatelessWidget implements PreferredSizeWidget {
+  static final double prefHeight = 80;
```

Comment:
What made you choose 80? Is that simply what is in the designs?

Reply:
yeah, this matched the designs